README.md

```diff
@@ -303,6 +303,7 @@ The result of a status check is the status object which contains SemVer strings
 }
 ```
 
+
```

Comment:
You mention that you _updated the documentation_, but there is just this empty line. Did you perhaps forget to commit the updated README?

Reply:
The README was updated in https://github.com/web-eid/web-eid.js/commit/de94cde5aa98d48ebd620720b81860d882320f0e, but then reverted it in the next commit and added the following point:
> * Reverted documentation change as the warnings are no longer related to the JS API